packages/config/config/devices/0x0159/smart_plug_16a.json

```diff
@@ -90,25 +91,198 @@
 		},
 		{
 			"#": "30",
-			"$import": "~/templates/master_template.json#state_after_power_failure_prev_off"
+			"label": "Restore on/off status",
+			"description": "After power failure",
+			"valueSize": 1,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Restores last on/off status",
+					"value": 0
+				},
+				{
+					"label": "Does not restore last on/off status",
+					"value": 1
+				}
+			]
 		},
 		{
 			"#": "40",
-			"label": "Power Consumption (Watt) Reporting Threshold",
+			"label": "Power Consumption Reporting Threshold",
+			"description": "Percentage of increase or decrease to be reported.",
 			"unit": "%",
 			"valueSize": 1,
 			"minValue": 0,
 			"maxValue": 100,
-			"defaultValue": 20
+			"defaultValue": 20,
+			"options": [
+				{
+					"label": "Disabled",
```

Comment:
This should actually be "Disable" for consistency with other files, not "Disabled"